samples/v3/translate_create_glossary.js

```diff
@@ -53,15 +53,22 @@ function main(
       glossary: glossary,
     };
 
-    // Create glossary using a long-running operation.
-    // You can wait for now, or get results later.
-    const [operation] = await translationClient.createGlossary(request);
-
-    // Wait for operation to complete.
-    await operation.promise();
```

Comment:
rather than taking this approach, you should be able to simply put a `try`/`catch` around `await operation.promise`; this should be equivalent to the `.catch` on the promise chain, and means we don't mix `async`/`await` and promises.

Reply:
The error that I'm specifically trying to catch (invalid projectId, glossary already exists, etc) happens at `const [operation] = await translationClient.createGlossary(request);`, so putting a try block around `operation.promise` doesn't end up catching it. 

Originally I put a `try/catch` around `const [operation] = await translationClient.createGlossary(request);`, which worked, but then I would need to terminate the program via `process.exit` because `throw` wouldn't suppress the unhandled promise rejection warning.

Alternatively, what would probably also work would be a single `try/catch` around the two await statements + the two `console.log` statements, but I also don't think that's great.

Reply:
I think part of the problem is that you call the outer `createGlossary`, which is going to create a promise that isn't handled (I'm talking about outside the internal `async` function`).

You could make the function `main()` async as well, and do something like this I believe:

```js
async main () {
  async createGlossary() {
   await foo
    await bar
  }

  try {
    await createGlossary();
  } catch (err) {
   console.info(err)
  }
}
```

Reply:
Just tried this and it does work, my only concern is that since we hide the `async main()`, then we now have a sample that is not runnable via a copy-paste of what's available on cloud.google.com.